metadata_layer.py

```diff
@@ -471,32 +869,25 @@ def __init__(
             contributors=None,
             measurements=None,
             links=None,
-            formats=None,
-            rights_uri=None,
-            last_update_time=None,
-            circulation=None,
+            provider_entry_updated=None,
```

Comment:
but data source can be something that's local to us, like the MetadataWrangler.  doesn't this property refer only to the last time we went over and checked the provider's remote server's idea of what the metadata should be?